docs/root/configuration/runtime.rst

```diff
@@ -4,8 +4,11 @@ Runtime
 =======
 
 The :ref:`runtime configuration <arch_overview_runtime>` specifies the location of the local file
-system tree that contains re-loadable configuration elements. If runtime is not configured, a "null"
-provider is used which has the effect of using all defaults built into the code.
+system tree that contains re-loadable configuration elements. Values can be viewed at the
+:ref:`/runtime admin endpoint <operations_admin_interface_runtime>`. Values can be modified and
+added at the :ref:`/runtime_modify admin endpoint <operations_admin_interface_runtime_modify>`. If
+runtime is not configured, an empty provider is used which has the effect of using all defaults
+built into the code, except for any values added via `/runtime_modify`.
```

Comment:
Maybe this could merit an additional ::attention:: comment on the dangers of the admin endpoint. We're now exposing a lot more knobs via this PR.